### ISSUE TITLE
Error in PhpStorm: java.lang.Throwable: Non-idempotent computation: it returns different results when invoked multiple times or on different threads:   1 != 2

### ISSUE THREAD

**Issue by manuglopez** [laravel-idea/plugin#321]
Here you are stacktrace

``` 
java.lang.Throwable: Non-idempotent computation: it returns different results when invoked multiple times or on different threads:
  1 != 2
  which is length of [com.laravel_idea.plugin.eloquent.b.d@4cce9a9a] and [dev.ekvedaras.laravelquery.reference.ColumnPsiReference(StringLiteralExpressionImpl: 'vip':(1,4)), com.laravel_idea.plugin.eloquent.b.d@6a5ef7c8]

Recomputation gives com.intellij.util.CachedValueBase$Data@309762ce which is equivalent to 'fresh'
Recomputation log:
  dev.ekvedaras.laravelquery.reference.ColumnReferenceProvider@11ecc6a8 returned [dev.ekvedaras.laravelquery.reference.ColumnPsiReference(StringLiteralExpressionImpl: 'vip':(1,4))]
  com.laravel_idea.plugin.a.b@15486265 returned [com.laravel_idea.plugin.eloquent.b.d@2d6a201c]
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:161)
	at com.intellij.util.IdempotenceChecker.reportFailure(IdempotenceChecker.java:90)
	at com.intellij.util.IdempotenceChecker.checkEquivalence(IdempotenceChecker.java:77)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:237)
	at com.intellij.psi.impl.PsiCachedValueImpl.getValue(PsiCachedValueImpl.java:43)
	at com.intellij.util.CachedValuesManagerImpl.getCachedValue(CachedValuesManagerImpl.java:72)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:155)
	at com.intellij.psi.util.CachedValuesManager.getCachedValue(CachedValuesManager.java:121)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:38)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:31)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.getReferences(StringLiteralExpressionImpl.java:432)
	at com.laravel_idea.plugin.extension.inspections.LaravelUnknownViewInspection$buildVisitor$1.visitPhpStringLiteralExpression(LaravelUnknownViewInspection.kt:32)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.accept(StringLiteralExpressionImpl.java:36)
	at com.jetbrains.php.lang.psi.elements.impl.PhpPsiElementImpl.accept(PhpPsiElementImpl.java:73)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:65)
	at com.intellij.codeInspection.InspectionEngine.createVisitorAndAcceptElements(InspectionEngine.java:56)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.runToolOnElements(LocalInspectionsPass.java:320)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$4(LocalInspectionsPass.java:279)
	at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:161)
	at com.intellij.util.AstLoadingFilter.forceAllowTreeLoading(AstLoadingFilter.java:153)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$5(LocalInspectionsPass.java:277)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:132)
	at com.intellij.util.AstLoadingFilter.disallowTreeLoading(AstLoadingFilter.java:121)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass.lambda$visitPriorityElementsAndInit$6(LocalInspectionsPass.java:277)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:136)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.concurrency.ApplierCompleter.execAndForkSubTasks(ApplierCompleter.java:149)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1096)
	at com.intellij.concurrency.ApplierCompleter.lambda$wrapInReadActionAndIndicator$1(ApplierCompleter.java:92)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:688)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:634)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:64)
	at com.intellij.concurrency.ApplierCompleter.wrapInReadActionAndIndicator(ApplierCompleter.java:104)
	at com.intellij.concurrency.ApplierCompleter.lambda$compute$0(ApplierCompleter.java:83)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:167)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:178)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:83)
	at java.base/java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:746)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```

**Comment by adelf** [comment]
It's a bug of the "Laravel Query" plugin. I'll try to suppress it. Thank you.

**Comment by manuglopez** [comment]
You are fast guy. OMG.

**Comment by adelf** [comment]
I haven’t released a fix yet:)

**Comment by manuglopez** [comment]
As I have read Implemented. I have closed the issue.

**Comment by adelf** [comment]
I’ll try to make an update next week

**Comment by adelf** [comment]
4.3.2 version is available.

**Comment by PhillipMwaniki** [comment]
Issue is still available in version 4.3.2.211

![image](https://user-images.githubusercontent.com/2037708/122191037-0c818680-ce9b-11eb-8656-165153fa5036.png)

**Comment by adelf** [comment]
I've created an issue for this plugin: https://github.com/ekvedaras/laravel-query-intellij/issues/42
